specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/framework/invalid.then.valid.credentials/request.rpt

```diff
@@ -0,0 +1,25 @@
+property invalidCredentials ${http:loginBase64Encoder("baduser:badpass")}
+property authHeader1 ${http:append("Basic ", invalidCredentials)}
+property goodCredentials ${http:loginBase64Encoder("joe:welcome")}
+property authHeader2 ${http:append("Basic ", goodCredentials)}
+connect http://localhost:8000/resource
+connected
+
+write method "GET"
+write header "Authorization" ${authHeader1}
+write close
+
+read status "401" "Unauthorized"
+read header "WWW-Authenticate" "Basic realm=" /.*/

```

Comment:
I would recommend grouping "Basic realm=" /.*/ into single value.
/Basic realm=.*/
I am not sure if we should use .* here. It basically means value of realm can be empty string. Need to check if that is the case in the RFC.